src/main/java/org/mitre/synthea/world/geography/CMSStateCodeMapper.java

```diff
@@ -246,6 +248,38 @@ public String zipToCountyCode(String zipcode) {
     return ssaTable;
   }
 
+  /**
+   * Get the FIPS county code for a given zipcode.
+   * @param zipcode the ZIP
+   * @return fips county code
+   */
+  public String zipToFipsCountyCode(String zipcode) {
+    // FIPS county codes are optional, so return blank if a match isn't found.
+    return fipsTable.getOrDefault(zipcode, "");
+  }
+
+  private HashMap<String, String> buildFipsTable() {
+    HashMap<String, String> fipsTable = new HashMap<String, String>();
+    List<LinkedHashMap<String, String>> csvData;
+    try {
+      String csv = Utilities.readResourceAndStripBOM("geography/fipscodes.csv");
+      csvData = SimpleCSV.parse(csv);
+    } catch (IOException e) {
+      throw new RuntimeException(e);
+    }
+    for (LinkedHashMap<String, String> row : csvData) {
+      String zipcode = row.get("zip");
+      if (zipcode.length() > 3) {
```

Comment:
This isn't needed, just pull out the nested if

Reply:
I fixed this, and added in another column in the carrier table.